tests/models/test_cli.py

```diff
@@ -116,6 +119,29 @@ def test_model_with_no_deployable_flavors_fails_pollitely():
         assert "No suitable flavor backend was found for the model." in stderr
 
 
+def test_serve_gunicorn_opts(iris_data, sk_model):
+    if sys.platform == "win32":
+        pytest.skip("This test requires gunicorn which is not available on windows.")
```

Comment:
Hm, this is a good point, do we need to document that `mlflow models serve` doesn't yet work on Windows?

Reply:
From offline discussion, sounds like we can add Windows support in the future

Reply:
As we discussed offline, we've never officially supported windows. The current level of support for windows only covers tracking.